storage/parquet/datasource_test.go

```diff
@@ -61,322 +76,315 @@ func TestParquetRecordStream_Get(t *testing.T) {
+					"int16_ct":                              -1,
+					"int16_lt":                              -1,
+					"int32_ct":                              -1,
+					"int32_lt":                              -1,
+					"int64_ct":                              -1,
+					"int64_lt":                              -1,
+					"int96_ct":                              0, // we're just taking first 8 bytes of a number
+					"uint8_ct":                              255,
+					"uint8_lt":                              255,
+					"uint16_ct":                             65535,
+					"uint16_lt":                             65535,
+					"uint32_ct":                             4294967295,
+					"uint32_lt":                             4294967295,
+					"uint64_ct":                             -1, // OctoSQL uses int64 under the hood, so this overflows.
+					"uint64_lt":                             -1,
+					"float_ct":                              -1.100000023841858, // why different decimals (?)
+					"float_lt":                              -1.100000023841858,
+					"double_ct":                             -1.1111111, // why different decimals (?)
+					"double_lt":                             -1.1111111,
+					"utf8":                                  "parquet00/",
+					"string":                                "parquet00/",
+					"10_byte_array_ct":                      []byte{255, 255, 255, 255, 255, 255, 255, 255, 255, 255},
+					"10_byte_array_lt":                      []byte{255, 255, 255, 255, 255, 255, 255, 255, 255, 255},
+					"date_ct":                               time.Date(1969, 12, 31, 0, 0, 0, 0, time.UTC),
+					"date_lt":                               time.Date(1969, 12, 31, 0, 0, 0, 0, time.UTC),
+					"decimal_int32_ct":                      -1e-05,
+					"decimal_int32_lt":                      -1e-05,
+					"decimal_int64_ct":                      -1e-10,
+					"decimal_int64_lt":                      -1e-10,
+					"decimal_byte_array_ct":                 -1e-2, // for some reason, encoding "ÿÿ" isn't working (?)
+					"decimal_byte_array_lt":                 -1e-2,
+					"decimal_flba_ct":                       -1e-5,
+					"decimal_flba_lt":                       -1e-5,
+					"enum_ct":                               base32.StdEncoding.EncodeToString([]byte("ENUM   000")),
+					"enum_lt":                               base32.StdEncoding.EncodeToString([]byte("ENUM   000")),
+					"time_millis_ct":                        time.Duration(0),
+					"time_utc_millis_lt":                    time.Duration(0),
+					"time_nonutc_millis_lt":                 time.Duration(0),
+					"time_micros_ct":                        time.Duration(0),
+					"time_utc_micros_lt":                    time.Duration(0),
+					"time_nonutc_micros_lt":                 time.Duration(0),
+					"time_utc_nanos":                        0,
+					"time_nonutc_nanos":                     0,
+					"timestamp_millis_ct":                   -1,
```

Comment:
why this and `timestamp_micros_ct` are still `-1` and not some Date?

Reply:
no idea